src/components/backupWallet/BackupWallet.js

```diff
@@ -74,6 +77,10 @@ const backupWalletStyles = ({ theme }) => ({
     minHeight: 100,
     marginBottom: theme.paddings.defaultMargin,
   },
+  resendButton: {
+    fontWeight: 'normal',
+    fontSize: normalize(15),
```

Comment:
Button seems to have a custom leter-spacing. It should be initial

![leter-spacing](https://user-images.githubusercontent.com/1731297/62973916-73d8d600-bded-11e9-8fac-344992c1c20b.gif)

Reply:
done!